changelogs/fragments/3655-use-publish_cmd.yaml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+    - ansible_galaxy_install - the output valeu ``cmd_args`` was bringing the intermediate command used to gather the state, instead of the command that actually performed the state change (https://github.com/ansible-collections/community.general/pull/3655).
+    - pipx - the output valeu ``cmd_args`` was bringing the intermediate command used to gather the state, instead of the command that actually performed the state change (https://github.com/ansible-collections/community.general/pull/3655).
```

Comment:
```suggestion
    - pipx - the output value ``cmd_args`` was bringing the intermediate command used to gather the state, instead of the command that actually performed the state change (https://github.com/ansible-collections/community.general/pull/3655).
```